### PR TITLE
Announce GA of memory control on stats cache

### PR DESCRIPTION
**Pull request by songrijie**
### What is changed, added or deleted? (Required)

[tidb_stats_cache_mem_quota](/system-variables.md#tidb_stats_cache_mem_quota-从-v610-版本开始引入) was generally available on v7.4.0. Include missing announcement in release notes.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/19226
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-7.4.0.md [hunk 1]
```diff
@@ -199,6 +199,12 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control on statistics cache is generally available [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    TiDB instances cache table statistics to accelerate execution plan generation, in order to improve SQL performance. Starting from v7.4.0, memory control on statistics cache becomes generally avaiable. By configuring system variable [`tidb_stats_cache_mem_quota`](/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), TiDB is able to limit the memory usage of the statistics cache. When the cache reaches the upper bound, TiDB automatically evicts inactive cache entries, helping to control instance memory usage and improve stability.
```
Comment (hfxsd) [inline releases/release-7.4.0.md:204]:
```suggestion
* Memory control on statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)

    TiDB instances cache table statistics to speed up execution plan generation and improve SQL performance. Starting with v7.4.0, memory control for the statistics cache is generally available (GA). By configuring the system variable [tidb_stats_cache_mem_quota](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```

File: releases/release-7.4.0.md [hunk 2]
```diff
@@ -199,6 +199,12 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control on statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    TiDB instances cache table statistics to speed up execution plan generation and improve SQL performance. Starting with v7.4.0, memory control for the statistics cache is generally available (GA). By configuring the system variable [tidb_stats_cache_mem_quota](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```
Comment (hfxsd) [inline releases/release-7.4.0.md:204]:
```suggestion
    TiDB instances cache table statistics to speed up execution plan generation and improve SQL performance. Starting from v7.4.0, memory control for the statistics cache is generally available (GA). By configuring the system variable [tidb_stats_cache_mem_quota](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```

File: releases/release-7.4.0.md [hunk 3]
```diff
@@ -199,6 +199,12 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control on statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    TiDB instances cache table statistics to speed up execution plan generation and improve SQL performance. Starting from v7.4.0, memory control for the statistics cache becomes generally available (GA). By configuring the system variable [tidb_stats_cache_mem_quota](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```
Comment (hfxsd) [inline releases/release-7.4.0.md:204]:
```suggestion
     Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). TiDB instances can cache table statistics to speed up execution plan generation and improve SQL performance. By configuring the system variable, you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
    
     Starting from v7.4.0, memory control for the statistics cache becomes generally available (GA).
```

File: releases/release-7.4.0.md [hunk 4]
```diff
@@ -199,6 +199,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control on statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), allowing TiDB instances to cache table statistics for faster execution plan generation and improved SQL performance. By configuring the system variable, you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
+
+    Starting from v7.4.0, memory control for the statistics cache becomes generally available (GA).
```
Comment (hfxsd) [inline releases/release-7.4.0.md:206]:
```suggestion
    Starting from v7.4.0, this feature becomes generally available (GA).
```

File: releases/release-7.4.0.md [hunk 5]
```diff
@@ -199,6 +199,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control on statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
```
Comment (hfxsd) [inline releases/release-7.4.0.md:202]:
```suggestion
* Memory control for the statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
```

File: releases/release-7.4.0.md [hunk 6]
```diff
@@ -199,6 +199,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control for the statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610), allowing TiDB instances to cache table statistics for faster execution plan generation and improved SQL performance. By configuring the system variable, you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```
Comment (hfxsd) [inline releases/release-7.4.0.md:204]:
```suggestion
     TiDB instances to cache table statistics for faster execution plan generation and improved SQL performance. Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). By configuring the system variable, you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```

File: releases/release-7.4.0.md [hunk 7]
```diff
@@ -199,6 +199,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control for the statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    TiDB instances can cache table statistics for faster execution plan generation and improved SQL performance. Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). By configuring the system variable, you can limit memory usage for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```
Comment (lilin90) [inline releases/release-7.4.0.md:204]:
```suggestion
    TiDB instances can cache table statistics to accelerate execution plan generation and improve SQL performance. Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). By configuring this system variable, you can set a memory usage limit for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```

File: releases/release-7.4.0.md [hunk 8]
```diff
@@ -199,6 +199,14 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-
 
     For more information, see [documentation](/system-variables.md#tidb_opt_enable_hash_join-new-in-v656-v712-and-v740).
 
+* Memory control for the statistics cache is generally available (GA) [#45367](https://github.com/pingcap/tidb/issues/45367) @[hawkingrei](https://github.com/hawkingrei)
+
+    TiDB instances can cache table statistics to accelerate execution plan generation and improve SQL performance. Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). By configuring this system variable, you can set a memory usage limit for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive entries, helping control instance memory usage and improve stability.
```
Comment (lilin90) [inline releases/release-7.4.0.md:204]:
```suggestion
    TiDB instances can cache table statistics to accelerate execution plan generation and improve SQL performance. Starting from v6.1.0, TiDB introduces the system variable [`tidb_stats_cache_mem_quota`](https://github.com/system-variables.md#tidb_stats_cache_mem_quota-new-in-v610). By configuring this system variable, you can set a memory usage limit for the statistics cache. When the cache reaches its limit, TiDB automatically evicts inactive cache entries, helping control instance memory usage and improve stability.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by hfxsd — APPROVED** [review]

**Pull request merged**